src/database/schema/ChatSchema.ts

```diff
@@ -179,19 +192,27 @@ class ChatSchema implements BaseDbSchema {
       user,
       status: dbObject.status,
       isMe: dbObject.is_me,
-      isContinuous: dbObject.is_continuous
+      isContinuous: dbObject.is_continuous,
+      attachmentJson
     });
   }
 
   static fromWebsocketObject(json): ChatSchema {
     let rawJson = null;
+    let attachmentJson = null;
 
     try {
       rawJson = JSON.stringify(json);
     } catch (e) {
       console.log('error stringify');
       console.log(e);
     }
+    try {
+      attachmentJson = JSON.stringify(json?.message?.attachments);
+    } catch (e) {
+      console.log('error stringify');
+      console.log(e);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In this section, you're stringifying `json` and `json?.message?.attachments` and catching any errors that occur. Similar to my previous comment, consider adding more context to the error logs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->